apps/server/src/modules/sharing/api/service/share-token-permission.service.spec.ts

```diff
@@ -57,7 +57,7 @@ describe('ShareTokenPermissionService', () => {
 				if (key === 'FEATURE_LESSON_SHARE') return false;
 				if (key === 'FEATURE_TASK_SHARE') return false;
 				if (key === 'FEATURE_COLUMN_BOARD_SHARE') return false;
-				if (key === 'FEATURE_ROOMS_ENABLED') return false;
+				if (key === 'FEATURE_ROOM_SHARE') return false;
```

Comment:
I don't understand why FEATURE_ROOM_SHARE needed to be added here?

Reply:
because it should have been here beforehand, as the service also checks for `FEATURE_ROOM_SHARE` and it wasnt tested here but only the removed env var